python/vyos/firewall.py

```diff
@@ -493,13 +518,15 @@ def geoip_update(firewall, force=False):
 
         # Map country codes to set names
         for codes, path in dict_search_recursive(firewall, 'country_code'):
-            set_name = f'GEOIP_CC_{path[1]}_{path[3]}'
-            if path[0] == 'name':
-                for code in codes:
-                    ipv4_codes.setdefault(code, []).append(set_name)
-            elif path[0] == 'ipv6_name':
-                for code in codes:
-                    ipv6_codes.setdefault(code, []).append(set_name)
+            set_name = f'GEOIP_CC_{path[1]}_{path[2]}_{path[4]}'
+            if path[1] == 'ipv6_name':
+                set_name = f'GEOIP_CC_name6_{path[2]}_{path[4]}'
+                if ( path[0] == 'ip' ) and ( path[1] == 'forward' or path[1] == 'input' or path[1] == 'output' or path[1] == 'name' ):
```

Comment:
Think this block needs to be unindented once?

Reply:
Indention of such section wasn't change in this commit/PR.. Only firewall path was change in order to match new syntax.

Reply:
Line 524 won't return True in any state as it's preceded by the check `if path[1] == 'ipv6_name':`